backend/tests/Functional/Controller/ShareControllerTest.php

```
@@ -2,50 +2,781 @@
+
+        $this->getJson('/api/shares/invitations/' . $firstToken);
+        self::assertResponseStatusCodeSame(410);
+
+        $this->getJson('/api/shares/invitations/' . $secondToken);
+        self::assertResponseIsSuccessful();
+    }
+
+    public function testInvitationsAreRateLimitedPerOwner(): void
+    {
+        $owner = $this->createAndLoginUser(['email' => 'prolific@test.local']);
+        $comic = ComicFactory::new()->ownedBy($owner)->create()->object();
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Tests\Factory\ComicFactory::new().


(staticMethod.notFound)